cli/src/program.rs

```diff
@@ -1478,6 +1515,72 @@ fn process_dump(
     }
 }
 
+fn process_dump_owned_accounts(
+    rpc_client: &RpcClient,
+    config: &CliConfig,
+    account_pubkey: Option<Pubkey>,
+    output_directory: &str,
+) -> ProcessResult {
+    if let Some(account_pubkey) = account_pubkey {
+        if let Some(account) = rpc_client
+            .get_account_with_commitment(&account_pubkey, config.commitment)?
+            .value
+        {
+            if account.owner == bpf_loader::id()
+                || account.owner == bpf_loader_deprecated::id()
+                || account.owner == bpf_loader_upgradeable::id()
+            {
+                let program_accounts = rpc_client.get_program_accounts(&account_pubkey);
+                match program_accounts {
+                    Ok(program_accounts) => {
+                        for (pubkey, account) in program_accounts {
+                            let output_path =
+                                format!("{}/{}.json", output_directory, pubkey.to_string());
```

Comment:
Can you use a `Path` and use `join` instead here? that way it'll work across platforms